extensions/Skyhigh173/bigint.js

```diff
@@ -0,0 +1,456 @@
+(function(Scratch){
+  'use strict';
+
+  const bi = x => {
+    try {
+      if (x.slice(-1) == 'n') {
+        return BigInt(x.slice(0, -1));
+      }
+      return BigInt(x);
+    } catch {
+      try {
+        return BigInt(Math.trunc(Number(x)));
```

Comment:
Might be a better idea to do Math.trunc by using string operations to remove the decimals, otherwise by converting to Number you can lose precision